sample/java/src/main/java/com/datadog/android/sample/SampleApplication.java

```diff
@@ -46,7 +46,7 @@ public void onCreate() {
 
         final DatadogConfig.Builder configBuilder;
         final String environment = "staging";
-        if (BuildConfig.DD_RUM_APPLICATION_ID != null) {
+        if (!BuildConfig.DD_RUM_APPLICATION_ID.isEmpty()) {
```

Comment:
I'd remove those checks to make the initialisation sequence more readable for customers looking at the sample. 
We can assume that the config is properly set up and add a note in the samples' readme on what to put in the "credentials/xxx" files

Reply:
Yup...this one I can remove...the other 3 regarding the endpoints override I need to keep otherwise it will override them with empty urls.